src/main/java/com/github/monster860/fastdmm/objtree/ObjectTreeParser.java

```diff
@@ -38,8 +39,10 @@
 	
 	public Map<String, String> macros = new HashMap<>();
 	public JFrame modalParent;
-	
-	public ObjectTreeParser() {
+
+    public static final CachedPattern quotesPattern = new CachedPattern("^\"(.*)\"$");
```

Comment:
This is only used within this class, no need to define it as `public static` also `final` modifiers should have it's variable named as `FULLCAPS`.

I'd suggest changing this to `private final CachedPattern QUOTES_PATTERN = new CachedPattern("^\"(.*)\"$");`